draft-ietf-quic-recovery.md

```diff
@@ -1023,15 +1027,13 @@ kLossReductionFactor:
   The RECOMMENDED value is 0.5.
 
 kPersistentCongestionThreshold:
-: Number of consecutive PTOs after which network is considered to be
+: Number of consecutive PTOs after which network might be considered to be
```

Comment:
```suggestion
: Number of consecutive PTOs after which the network is considered to be
```

I prefer "is" instead of "might be" given you already have considered.

Reply:
I don't want to say "is" because that implies that persistent congestion has happened. As Marten notes, that is only done when the ACK is received.